mumblepahelper.cpp

```diff
@@ -162,3 +162,44 @@ void MumblePAHelper::on_action_Quit_triggered(bool) {
 void MumblePAHelper::on_action_Rescan_triggered(bool) {
     plugins->rescanPlugins();
 }
+
+void MumblePAHelper::on_action_SystemPluginsLocation_triggered(bool) {
+	QString SystemPluginsLocation = QFileDialog::getExistingDirectory(this, tr("Select system plugins directory"), plugins->qsSystemPlugins, QFileDialog::ShowDirsOnly | QFileDialog::DontResolveSymlinks);
+	if (SystemPluginsLocation != NULL)
+		plugins->qsSystemPlugins = SystemPluginsLocation;
+	plugins->rescanPlugins();
+}
+
+void MumblePAHelper::on_action_UserPluginsLocation_triggered(bool) {
+    QString UserPluginsLocation = QFileDialog::getExistingDirectory(this, tr("Select user plugins directory"), plugins->qsUserPlugins, QFileDialog::ShowDirsOnly | QFileDialog::DontResolveSymlinks);
```

Comment:
The text in the dialog ("Select user plugins directory") should match the text on the action/menu item.
So this should probably read "Set user plugin directory"...